packages/iTwinUI-react/src/core/Toast/ToastWrapper.test.tsx

```diff
@@ -5,7 +5,7 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 
-import { ToastWrapper } from './ToastWrapper';
+import { ToastWrapper, ToastWrapperHandle } from './ToastWrapper';
 import { ToastProps } from './Toast';
 import { act } from 'react-dom/test-utils';
```

Comment:
Why act is imported from here???

Reply:
But removed all use cases of that import..